### PR TITLE
add alert

### PR DESCRIPTION
**Pull request by harshkotkar5557**

### CODE REVIEW

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    gap: 20px;
+}
+
+.alert-success,.alert-danger,.alert-info,.alert-warning{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```
Comment (vaibhav18matere) [inline alert/alert.css:14]: you can make separate class for this, everything else looks good !

**Review by vaibhav18matere — APPROVED** [review]

File: alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    border-left: 5px solid var(--darkYellow);
+    background-color: var(--warning);
+    color: var(--darkYellow);
+}
+.close {
+    font-size: 2rem;
+}
+
+.middle-text {
+    line-height: 25px;
```
Comment (aayushjain-code) [inline alert/alert.css:49]: you can make px --> rem else looks good to me

File: alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    flex-wrap: wrap;
+    justify-content: space-between;
+    gap: 20px;
+}
+
+.alert-success,.alert-danger,.alert-info,.alert-warning{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2px 12px;
```
Comment (MuskanChhatrasal) [inline alert/alert.css:15]: You can have pre-defined class for margins and paddings. Else LGTM!!

**Review by MuskanChhatrasal — APPROVED** [review]

File: alert/alert.css (hunk 1 above)
Comment (harshkotkar5557) [reply alert/alert.css:14]: Sure I will note down it and refactor it at final stage

File: alert/alert.css (hunk 2 above)
Comment (harshkotkar5557) [reply alert/alert.css:49]: Okay I will do it Thanks

File: alert/alert.css (hunk 3 above)
Comment (harshkotkar5557) [reply alert/alert.css:15]: Sure I will do it in final refactor. Thanks

**Pull request merged**